twitter.py

```diff
@@ -21,3 +21,23 @@ def post_tweet(tweet: str):
         print('Successfully Tweeted: ' + tweet)
     except tweepy.error.TweepError as e:
         print(e)
+
+def post_thread(tweets: List[str]):
+    """
+    Posts a thread of tweets, received as a list
+    of strings that are less than 280 characters each.
+    """
+
+    api = _get_api()
+    try:
+        last_status_id = None
+        for tweet in tweets:
+            if last_status_id is not None:
+                status = api.update_status(tweet)
+                print('Successfully Tweeted: ' + tweet + ' with id ' + str(status.id))
+            else:
+                status = api.update_status(tweet, last_status_id)
+                print('Successfully Tweeted: ' + tweet + ' with id ' + str(status.id) + " as a reply to tweet with id " + str(last_status_id))
```

Comment:
I didn't expect any contributors to this repo so I have not mentioned contributing guidelines but please try to follow pep8 style and keep maximum line length less than 80.

Reply:
I have pushed a commit that reduces the line length overall to less than 80! It's `24feef9`.